### PR TITLE
adding bugfix for going back in replay mode

### PR DESCRIPTION
**Pull request by jyoo980**
I should not be allowed to write programs

### CODE REVIEW

File: provdebug/pvdebug.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def run():
                     debugTrace.current_record().prettyPrint(frame_length)
                 continue
             elif userFlag == "b" or userFlag == "back":
-                record = debugTrace.next_record()
+                record = debugTrace.prev_record()
```
Comment (jyoo980) [inline provdebug/pvdebug.py:110]: I'm a monkey

Comment (alison-li) [reply provdebug/pvdebug.py:110]: Is that why pressing back sometimes didn't actually show the previous program? :o

Comment (alison-li) [reply provdebug/pvdebug.py:110]: Good catch anyhow, lol

**Comment by alison-li** [comment]
(oops I just merged it right away before approving)

**Review by alison-li — COMMENTED** [review]
excellent

**Pull request merged**